src/components/promptBuilder/promptBuilder.stories.tsx

```diff
@@ -0,0 +1,327 @@
+/* eslint-disable no-magic-numbers */
+
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Chip from '@mui/material/Chip'
+import Divider from '@mui/material/Divider'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import { Stack } from '@mui/system'
+import type { Meta, StoryFn } from '@storybook/react/*'
+import { Server } from 'mock-socket'
+import React, { useState } from 'react'
+import { v4 as getUUID } from 'uuid'
+
+import TextInput from '../input/textInput/textInput'
+import MessageSpace from '../messageSpace/messageSpace'
+import Question from '../question/question'
+import Text from '../text/text'
+import type { Message, QuestionProps } from '../types'
+import { getMockWebSocketClient, sendMessageToClient } from './mockWebSocket'
+import PromptBuilder from './promptBuilder'
+
+const meta: Meta<React.ComponentProps<typeof PromptBuilder>> = {
+  title: 'Rustic UI/Prompt Builder/Prompt Builder',
+  component: PromptBuilder,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'centered',
+  },
+  decorators: [
+    (Story: StoryFn) => {
+      return (
+        <div style={{ width: 'clamp(300px,50vw,600px)' }}>
+          <Story />
+        </div>
+      )
+    },
+  ],
+}
+
+meta.argTypes = {
+  ws: {
+    description:
+      'WebSocket connection to send and receive messages to and from a backend. The key methods to focus on are `send` and `onReceive`.\n\n' +
+      "The `onReceive` method is responsible for providing access to the WebSocket's `onmessage` event through `handler`, allowing `PromptBuilder` to receive and render messages from the server. An `onReceive` method is needed for the functionality of this component.\n\n" +
+      'The `send` method is responsible for sending user inputs to the server. It sends the captured inputs as messages whenever the "Next Question". It will also send captured inputs when clicking "Generate", even if "Next Question" was not clicked beforehand.',
```

Comment:
```suggestion
      'The `send` method is responsible for sending user inputs to the server. It sends the captured inputs as messages whenever the "Next Question" is clicked. It will also send captured inputs when clicking "Generate", even if "Next Question" was not clicked beforehand.',
```